### ISSUE TITLE
Chrome browser getting 302 error ERR_TOO_MANY_REDIRECTS

### ISSUE THREAD

**Issue by w0nd3r365** [jellyfin/jellyfin.org#813]
I exactly followed as documented the example jellyfin [nginx config](https://jellyfin.org/docs/general/networking/nginx/#nginx-with-subpath-exampleorgjellyfin) for jellyfin with subpath but I couldn't load the nginx proxied jellyfin page in chrome, getting the error ERR_TOO_MANY_REDIRECTS.

There's an error in the current https://jellyfin.org/docs/general/networking/nginx/ as documented in [issue #6610](https://github.com/jellyfin/jellyfin/issues/6610) in jellyfin repo, fixed by [comment from CaptainXLAB](https://github.com/jellyfin/jellyfin/issues/6610#issuecomment-1675311204). I'll provide more info if needed.

**Comment by felix920506** [comment]
Hi, the best way to get this addressed would be to edit the file and submit a PR.

The file for nginx is this one: https://github.com/jellyfin/jellyfin.org/blob/master/docs/general/networking/nginx.md

**Comment by JPVenson** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by w0nd3r365** [comment]
After much testing I've come to realize that the endless redirect loop issue for Nginx with subpath was rather simple to fix. It was caused by the trailing `/jellyfin/` in the proxy_pass directive. I've filed PR #830  that fixes it.

Why does this happen? Normally in Nginx if we put trailing slash `/` at the end of the `proxy_pass` directive with a variable, it will ONLY proxy `/` to jellyfin server without the entire URI. Jellyfin then redirects to `/jellyfin` and nginx redirects `/jellyfin` to `/jellyfin/` location block which contains the which contains the problematic trailing slash in the `proxy_pass` directive causing the redirection cycle to begin again, endlessly. (source: https://distinctplace.com/2017/04/19/nginx-resolver-explained/) 

The potential problem with the solution proposed in this [comment](https://github.com/jellyfin/jellyfin.org/issues/813#issuecomment-1894747531) is explained in https://distinctplace.com/2017/04/19/nginx-resolver-explained/ which basically says that when you specify hostname for `proxy_pass` rather than a static IP address, Nginx will resolve it only once on startup or reload and then cache resulting IP address indefinitely until it's restarted or reloaded. This becomes problematic if the IP address of the jellyfin hostname changes frequently, because then the Nginx-cached IP address would be invalid and may end up proxying some other site or replying 502 bad gateway. If using a static IP address, or a hostname with a static IP address, then the solution would work.
